eng/pipelines/windows.yml

```diff
@@ -115,14 +115,27 @@ stages:
         buildExtraArguments: /p:RuntimeOS=win10
 
         variables:
-          - uapNetfxQueues: Windows.10.Amd64.ClientRS5.Open
-          - windowsArmQueue: Windows.10.Arm64.Open
+          - ${{ if eq(parameters.isInternalBuild, 'false') }}:
+            - nanoQueues: "`(Windows.Nano.1809.Amd64.Open`)windows.10.amd64.serverrs5.open@mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1809-helix-amd64-08e8e40-20200107182504"
+            - uapNetfxQueues: Windows.10.Amd64.ClientRS5.Open
+            - windowsArmQueue: Windows.10.Arm64.Open
+
+            - ${{ if eq(parameters.fullMatrix, 'false') }}:
+              - netcoreappWindowsQueues: Windows.7.Amd64.Open+Windows.81.Amd64.Open+Windows.10.Amd64.Client21H1.Open
```

Comment:
Before had an ES build, which I assume means the OS is in Spanish.  After does not.

Reply:
There's no client queue with ES anymore

Reply:
That's... unfortunate.  But, if that's just where things are, then I guess we have to take what we can get.